sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryHttpPolicy.java

```diff
@@ -56,6 +57,10 @@ public HttpPipelinePolicy create() {
 
     @Override
     public Mono<HttpResponse> process(HttpPipelineCallContext context, HttpPipelineNextPolicy next) {
+        if ((boolean) context.getData(DISABLE_TRACING_KEY).orElse(false)) {
```

Comment:
This would only make it in the next release now?

Reply:
Yes, this is for next release only.